packages/design-system/src/components/SkipNav/SkipNav.stories.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+import React from 'react';
+
+import SkipNav from './SkipNav';
+
+export default {
+  title: 'Components/Skip Nav',
+  component: SkipNav,
```

Comment:
I'm seeing the console error 
```
The prop `href` is marked as required in `SkipNav`
```
Consider resolving this with a default arg passed to the SkipNav like
```
args: {
    href: '#'
  }
```